docs/docs-content/clusters/edge/site-deployment/site-installation/cluster-deployment.md

```diff
@@ -119,8 +130,22 @@ To learn more, check out the resource from the etcd documentation titled [Why an
 
 7. Navigate to the **Edge Hosts drop-down Menu** and select your Edge host.
 
+8. (Optional) When you select your Edge host, you can optionally specify a static IP address for the Edge host. If you want to specify a static IP, toggle on **Static IP** and provide the following information:
+    
+  | **Field** | **Description** |
+  |-------|-------------|
+  | IP Address | The static IP address assigned to your Edge host. This should be unique within your network. |
+  | Default gateway | The IP address of the default gateway for your cluster network. This gateway routes traffic from your cluster to external networks, including the internet. |
+  | Subnet mask | The subnet mask of your cluster network. This defines the range of IP addresses within your cluster network. |
+  | DNS server | The IP address of the DNS server your cluster uses for domain resolution. | 
+
+  If certain network information is already available, the corresponding fields will be pre-populated. 
+  
+:::caution
+  After you add the Edge host to your cluster, you will not be able to change its IP address unless you remove and re-add them back to the cluster. 
+:::
```

Comment:
Lenny, play with this to get it lined up with the bullet. You may have to tab the whole component, or create a `br /` and then add the element below the breakline followed with a tab.

![CleanShot 2023-10-30 at 14 26 02](https://github.com/spectrocloud/librarium/assets/29551334/704ee681-00de-4f48-9549-601ac5ff40f2)